### DISCUSSION TITLE
Chromosome type

### DISCUSSION THREAD

**Discussion by sudistes4** [ahmedfgad/GeneticAlgorithmPython#259] [Q&A]
Hello !
I am working on a school project. I have difficulties to determinate the appropriate type for my chromosomes.

For the moment, our chromosome consists of a list of lists (each gene is a list containing characteristics which are strings).
I have the following issue : 
ValueError: A 2D list is expected to the initail_population parameter but a (3-D) list found.

Is there any way to solve it ?

Thanks for your help

**Comment by ahmedfgad** [comment]
Hi @sudistes4,

The population must be 2D. You can build some functionalities that convert your format to a format compatible with PyGAD.

**Reply by sudistes4** [reply]
Ok, thanks !
